session1/examples/interface.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+exports.__esModule = true;
+var firstName = "John";
+var lasttName = "Doe";
+var age = 22;
+var address = {
+    apartment: 101,
+    street: "Fort Street",
+    city: "Mumbai",
+    pin: 400001,
+    state: "Maharashtra"
+};
+var getAddress = function (address) {
+    var addressString = "Apartment: " + address.apartment + ", Street:  " + address.street + ", City: " + address.city + ", PIN: " + address.pin + ", State: " + address.state;
```

Comment:
Use template literals in ES6 and not ES5 syntax.

Reply:
TBD